tests/installation/ipxe_install.pm

```diff
@@ -124,6 +159,11 @@ END_BOOTSCRIPT
 
     my $response = HTTP::Tiny->new->request('POST', $url, {content => $bootscript, headers => {'content-type' => 'text/plain'}});
     diag "$response->{status} $response->{reason}\n";
```

Comment:
Suggest to add the code as following to checking if IPXE_HTTPSERVER works
```    die "Fail to post IPXE_HTTPSERVER\n" unless $response->{success};```

Reply:
Added. Trying out to verify...

Reply:
This is fixed in #17583. We could merge that first and rebase this on top.

Reply:
Sounds great. I remove the change from my PR.